back/src/main/resources/properties/crypto.yml

```
@@ -13,3 +17,6 @@ verification-token:
   aes-secret: ${EMAIL_TOKEN_AES_SECRET}
   hmac-secret: ${EMAIL_TOKEN_HMAC_SECRET}
   expiration: ${EMAIL_TOKEN_EXPIRATION}
+
+db:
+  aes-secret: ${TEST_DB_AES_SECRET}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)